SConstruct

```diff
@@ -136,7 +136,7 @@ opts.Add(EnumVariable('bits', "Target platform bits", 'default', ('default', '32
 opts.Add('p', "Platform (alias for 'platform')", '')
 opts.Add('platform', "Target platform (%s)" % ('|'.join(platform_list), ), '')
 opts.Add(EnumVariable('target', "Compilation target", 'debug', ('debug', 'release_debug', 'release')))
-opts.Add('tools', "Build the tools a.k.a. the Godot editor (yes/no)", 'yes')
+opts.Add(BoolVariable('tools', "Build the tools a.k.a. the Godot editor", True))
```

Comment:
You missed those ones: https://github.com/QuLogic/godot/blob/aafd2cd889c0ecb40db7890c2e7beb0501fcebe9/modules/squish/config.py#L9
https://github.com/QuLogic/godot/blob/aafd2cd889c0ecb40db7890c2e7beb0501fcebe9/modules/tinyexr/config.py

And possibly a couple others. Otherwise, looks great :)

Reply:
Ah, I didn't know there was a `config.py` also; I'll try to see if I can find those too.

Not sure why GitHub thinks this is outdated since I've only pushed once.

Reply:
> Not sure why GitHub thinks this is outdated since I've only pushed once.

Because subsequent commits change things in the same code chunk, so as far as GitHub knows the line I commented on may no longer be the same in your branch's HEAD.

Reply:
Ah, you commented on the commit view, not the overall one.